source/guide_roundcube.rst

```diff
@@ -0,0 +1,222 @@
+.. highlight:: console
+
+.. author:: Achim | pxlfrk <hallo@pxlfrk.de>
+
+.. tag:: lang-php
+.. tag:: web
+.. tag:: mail
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/roundcube.svg
+      :align: center
+
+#################
+Roundcube Webmail
```

Comment:
While roundcube itself isn't terribly consistent about that, the project seems to be named "Roundcube" not "Roundcube Webmail". Please adapt this, to keep the our guide list clean. Feel free to add `webmail` to the tag list to make it easier to discover, though!

Reply:
I changed the wording in the guide to `Roundcube`.